src/base/Stopwatch.hh

```diff
@@ -34,7 +34,7 @@ class Stopwatch
     inline Stopwatch();
 
     // Get the current elapsed time [s]
-    inline real_type operator()() const;
+    inline double operator()() const;
```

Comment:
Could enhance the git log to record the reason behind this change?

Reply:
Yeah. Basically this is a CPU-only operation for testing/diagnostics, not related to any physics or on-device code, so we should use the "native" higher-precision type.